repository-data/application/src/main/resources/hcm-config/namespaces/hee/publicationLandingPage.yaml

```diff
@@ -160,12 +160,163 @@ definitions:
             hipposysedit:primary: false
             hipposysedit:type: Taxonomy
             hipposysedit:validators: [required]
+        /hipposysedit:nodetype[2]:
```

Comment:
@FranciscoRuizHEE - Could you remove the uncommitted changes from this file as I assume we are not making any changes to the `Publication landing page` document type as part of this work ?